TurkishFilter/sections/specific.txt

```diff
@@ -122,6 +122,20 @@ aykutozdemir.com.tr##.theiaStickySidebar > #text-4
 aykutozdemir.com.tr##.theiaStickySidebar > #text-5
 aykutozdemir.com.tr##.theiaStickySidebar > #text-10
 7deniz.net###lock > div[style] > a[href][target="_blank"] > img
+uludagsozluk.com###overlay
+uludagsozluk.com##.popup
+uludagsozluk.com###main > a[href*="bit.ly"][target="_blank"]
```

Comment:
It's better to exchange this rule to the common `##a[href^="https://bit.ly/"] > img`

Reply:
Also we have a similar rule `kofdizi.com###overlay` for `#overlay`. Add a domain to them.

Reply:
Suggested rule will leave 20px gap in website layout wich could be considered as an ad leftover.
I am targeting <a> tag instead to go seamless.

Reply:
Ok, then you can throw away the part `img` and sorte similar rules to one common.